apps/meteor/client/views/admin/viewLogs/hooks/useAnalyticsObject.ts

```diff
@@ -0,0 +1,8 @@
+import { useEndpoint } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+
+export const useAnalyticsObject = () => {
+	const getAnalytics = useEndpoint('GET', '/v1/statistics');
+
+	return useQuery(['analytics'], () => getAnalytics({}));
+};
```

Comment:
don't return the entire useQuery return, you can return only the necessary objects, and also you can get rid of `data` and send only the payload

Reply:
If I do this react query will stop interacting correctly with itself and cause TS errors and, consequently, unnecessary overhead, also will make future usage of this hook harder than it needs to be.

Reply:
If I do this react query will stop interacting correctly with itself and cause TS errors and, consequently, unnecessary overhead, also will make future usage of this hook harder than it needs to be.

Reply:
For an example:
![image](https://github.com/RocketChat/Rocket.Chat/assets/43561537/5641162f-3615-4e69-91b6-9ea03e42b518)
![image](https://github.com/RocketChat/Rocket.Chat/assets/43561537/a60b82e6-dceb-424f-9f99-0dd937c837b4)